extensions/wikia/Search/classes/Result.php

```diff
@@ -64,78 +76,91 @@ public function getText( $field = 'text', $wordLimit = null ) {
-		$textAsString = $this->removePrefix($this->findFilePrefix($this->getTitle(false)), $textAsString);
+		$textAsString = $this->removePrefix( $this->findFilePrefix( $this->getTitle( false ) ), $textAsString );
+
 		return $textAsString;
 	}
 
 	/**
 	 * Get the hub name, translated to content language
+	 *
 	 * @return string
 	 */
 	public function getHub() {
-		return wfMessage('hub-'.$this->getText( 'hub_s' ))->text();
+		return wfMessage( 'hub-' . $this->getText( 'hub_s' ) )->text();
 	}
 
 	/**
 	 * Returns the string value of the document's title
+	 *
 	 * @see    WikiaSearchResultTest::testTitleFieldMethods
 	 * @return string
 	 */
-	public function getTitle($removeFilePrefix = true) {
+	public function getTitle( $removeFilePrefix = true ) {
 		$result = '';
-		if ( isset( $this->_fields[Utilities::field('title')] )  ) {
-			$result = $this->_fields[Utilities::field('title')];
+		if ( isset( $this->_fields[Utilities::field( 'title' )] ) ) {
+			$result = $this->_fields[Utilities::field( 'title' )];
 		} else if ( isset( $this->_fields['title'] ) ) {
 			$result = $this->_fields['title'];
-		} else if ( isset( $this->_fields[Utilities::field('title', 'en')] )  ) { // for video wiki
-			$result = $this->_fields[Utilities::field('title', 'en')];
+		} else if ( isset( $this->_fields[Utilities::field( 'title', 'en' )] ) ) { // for video wiki
+			$result = $this->_fields[Utilities::field( 'title', 'en' )];
 		}
-		if ($removeFilePrefix) {
-			$result = $this->removePrefix($this->findFilePrefix($result), $result);
+		if ( $removeFilePrefix ) {
+			$result = $this->removePrefix( $this->findFilePrefix( $result ), $result );
 		}
+
 		return $result;
 	}
 
-	public function removePrefix($prefix, $value) {
-		if ($prefix && (strpos($value, $prefix) === 0)) {
-			return substr($value, strlen($prefix));
+	public function removePrefix( $prefix, $value ) {
+		if ( $prefix && ( strpos( $value, $prefix ) === 0 ) ) {
+			return substr( $value, strlen( $prefix ) );
 		} else {
 			return $value;
 		}
 	}
 
 	/**
 	 * Find the file prefix in title, if result is in File namespace.
-	 * @param  string $value
+	 *
+	 * @param string $title
+	 *
 	 * @return string
 	 */
-	public function findFilePrefix($title) {
-		if (!empty($this->_fields['ns']) && $this->_fields['ns'] == \NS_FILE && strpos($title, ":") !== false) {
+	public function findFilePrefix( $title ) {
+		if ( !empty( $this->_fields['ns'] ) && $this->_fields['ns'] == \NS_FILE && strpos( $title, ":" ) !== false ) {
 			/**
 			 * find 'File:' prefix (in content language) in title
 			 * we could try to use Title class or wgContLang->getNsText here, but none of those actually
 			 * will allow us to get potentially i18n'ed namespace prefix in a simple and working way, while
 			 *a simple explode with limit will work
 			 */
-			list ($prefix, $rest) = explode(":", $title, 2);
+			list ( $prefix, $rest ) = explode( ":", $title, 2 );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$rest" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)